### PR TITLE
Added new page_for_expected_runtime option

### PR DESCRIPTION
**Pull request by solarkennedy**
This is just a first pass at this.

I would like to not change any other behavior in the monitoring script yet, just add the additional option.

### CODE REVIEW

**Comment by qui** [comment]
Do you need to do this, too? 

https://github.com/Yelp/Tron/blob/0ada41a2cd2d4051ae25b1b5d92f2de978bbc92b/tron/bin/check_tron_jobs.py#L396

**Comment by solarkennedy** [comment]
> Do you need to discard the arg, like this?
> 
> https://github.com/Yelp/Tron/blob/0ada41a2cd2d4051ae25b1b5d92f2de978bbc92b/tron/bin/check_tron_jobs.py#L396-L398

Oh right, ty

File: tron/bin/check_tron_jobs.py [hunk 1]
```diff
@@ -124,8 +124,13 @@ def compute_check_result_for_job_runs(client, job, job_content, url_index, hide_
         status = 0
         stderr = ""
     elif last_state == State.STUCK:
-        prefix = f"WARN: Job {job_run_id} exceeded expected runtime or still running when next job is scheduled on {cluster}"
-        status = 1
+        if job['monitoring'].get("page_for_expected_runtime", False):
```
Comment (qui) [inline tron/bin/check_tron_jobs.py]: I think this conditional is the opposite of what you want?

Comment (solarkennedy) [reply tron/bin/check_tron_jobs.py]: 😬

**Comment by qui** [comment]
er I guess I should be asking for tests... I'll do that on the next review

**Pull request merged**